### PR TITLE
build: add use-dusk option

### PR DESCRIPTION
**Pull request by alichraghi**
Related to https://github.com/hexops/mach-dusk/pull/75

- [x] By selecting this checkbox, I agree to license my contributions to this project under the license(s) described in the LICENSE file, and I have the right to do so or have received permission to do so by an employer or client I am producing work for whom has this right.

### CODE REVIEW

File: examples/rotating-cube/main.zig [hunk 1]
```diff
@@ -158,7 +159,7 @@ pub fn update(app: *App) !bool {
         const ubo = UniformBufferObject{
             .mat = zm.transpose(mvp),
         };
-        encoder.writeBuffer(app.uniform_buffer, 0, &[_]UniformBufferObject{ubo});
+        queue.writeBuffer(app.uniform_buffer, 0, &[_]UniformBufferObject{ubo});
```
Comment (pdoane) [inline examples/rotating-cube/main.zig:162]: We may want to leave this as-is?  Although it's fine with me to change as other examples exercise the encoder writeBuffer method.

Comment (alichraghi) [reply examples/rotating-cube/main.zig:162]: it's a dawn extension and should be avoided as we don't know if they are going to keep it

Comment (emidoots) [reply examples/rotating-cube/main.zig:162]: Interesting, I didn't know this. Which one is the extension?

**Review by emidoots — APPROVED** [review]

File: examples/rotating-cube/main.zig (hunk 1 above)
Comment (alichraghi) [reply examples/rotating-cube/main.zig:162]: `CommandEncoder.writeBuffer`

**Pull request merged**